Makefile

```diff
@@ -20,6 +20,7 @@ endif
 GO 			:= GOOS=$(OS) GOARCH=$(ARCH) go
 GOGETTER	:= GOPATH=$(shell pwd)/.tmpdeps go get -d
 GOLDFLAGS	:= -ldflags "-X main.version=$(BUILDREV)"
+PWD := $(shell pwd)
```

Comment:
I don't have a windows computer, but this is probably the only non-portable part of this PR.